File: fib.exs

defmodule Fibonacci do
  def fib(n) when n < 2 do
      n
  end

  def fib(n) do
      fib(n - 1) + fib(n - 2)
  end
end

if System.argv != [] do
  IO.puts Fibonacci.fib(String.to_integer(List.first(System.argv())))
end

File: fib.cc
#include <iostream>
#include <string>

int fib(int n) {
  return n < 2 ? n : fib(n - 1) + fib(n - 2);
}

int main(int argc, char** argv) {
  std::cout << fib(std::stoi(argv[1])) << std::endl;
  return 0;
}
